cli/flags.go

```diff
@@ -734,6 +734,48 @@ func (f *FlagSection) Uint64Var(i *Uint64Var) {
 	})
 }
 
+// LoggerConfigurer is the interface to configure a logger.
+type LoggerConfigurer interface {
+	SetLevel(level string)
+	SetMode(mode string)
+}
+
+// LoggerVar binds common logging flags to a [LoggerConfigurer].
+func (f *FlagSection) LoggerVar(i LoggerConfigurer) {
+	parser := func(s string) (string, error) { return s, nil }
+	printer := func(v string) string { return v }
+	var dummy string
+
+	Flag(f, &Var[string]{
+		Name:    "log-level",
+		Usage:   "To specify the log level with one of debug|info|warn|error|fatal",
+		Example: "info",
+		Default: "warn",
+		EnvVar:  "LOG_LEVEL",
+		Predict: predict.Set{"debug", "info", "warn", "error", "fatal"},
```

Comment:
Not for this PR, but as we think about the migration to slog, it would be nice if these value came from constants in the logging package.